src/AltinnCore/Designer/Controllers/ManualTestingController.cs

```diff
@@ -231,13 +235,13 @@ public async Task<IActionResult> LoginTestUser(string org, string service, int i
                         AllowRefresh = false,
                     });
 
-            List<Reportee> reporteeList = _authorization.GetReporteeList(profile.UserId);
-            Reportee reporteeBE = null;
+            List<Party> reporteeList = _authorization.GetPartyList(profile.UserId);
+            Party reporteeBE = null;
 
-            if (!string.IsNullOrEmpty(reportee) && reporteeList.Any(r => r.ReporteeNumber.Equals(reportee)))
+            if (!string.IsNullOrEmpty(reportee) && reporteeList.Any(r => (r.PartyTypeName == PartyType.Person) ? r.SSN.Equals(reportee) : r.OrgNumber.Equals(reportee)))
             {
-                reporteeBE = reporteeList.FirstOrDefault(r => r.ReporteeNumber.Equals(reportee));
-                HttpContext.Response.Cookies.Append("altinncorereportee", reporteeBE.PartyID.ToString());
+                reporteeBE = reporteeList.FirstOrDefault(r => (r.PartyTypeName == PartyType.Person) ? r.SSN.Equals(reportee) : r.OrgNumber.Equals(reportee));
+                HttpContext.Response.Cookies.Append("altinncorereportee", reporteeBE.PartyId.ToString());
```

Comment:
what are the consequences of changing the cookiename to AltinnParty? Do we know? I think we should eliminate the use of the word reportee where this is possible, to make it easier to read and understand the code

Reply:
Don't know. I will test it out